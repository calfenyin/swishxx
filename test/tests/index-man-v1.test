index | | -e man:*.[1-9] -v1 | . | 0
