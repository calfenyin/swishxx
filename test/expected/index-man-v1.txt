
index: done:
  4 indexed
  4878 words, 1838 indexed, 487 unique

